src/main/scala/Keys.scala

```diff
@@ -28,7 +28,7 @@ object Keys {
   def kotlinPlugin(name: String) = sbt.Keys.libraryDependencies +=
     "org.jetbrains.kotlin" % ("kotlin-" + name) % kotlinVersion.value % "compile-internal"
 
-  def kotlinClasspath(config: Configuration, classpathKey: TaskKey[sbt.Keys.Classpath]): Setting[_] =
+  def kotlinClasspath(config: Configuration, classpathKey: SettingKey[sbt.Keys.Classpath]): Setting[_] =
```

Comment:
oh, I was expecting that there is some class that is parent for TaskKey and SettingKey and missed Initialize